app/core/serializers.py

```diff
@@ -0,0 +1,43 @@
+from rest_framework.serializers import ModelSerializer, Serializer
+
+from core.models import User, Event
+
+
+class HomeSerializer(Serializer):
+    class Meta:
+        model = Event
+        fields = (
+            'name',
+            'organizer',
+            'tags',
+            'date',
+        )
+
+
+class EventSerializer(ModelSerializer):
+    class Meta:
+        model = Event
+        fields = (
+            'name',
+            'organizer',
+            'guests',
+            'tags',
+            'attachment',
+            'date',
+            'notes',
+        )
+
+
+class UserSerializer(ModelSerializer):
```

Comment:
set date_joined() to current date only while save()

Reply:
also exclude fields like is_staff from editing coz shit can get wild

Reply:
> set date_joined() to current date only while save()

why?